misc/cfg/gfx_gl_faithful.cfg

```diff
@@ -156,10 +156,12 @@ gl_colorlights               "0"
 
 //Screen & Powerup Blends
 cl_bonusflash                "0"
+cl_demoplay_flash            "1"
 gl_cshiftpercent             "100"
 gl_hwblend                   "1"
 gl_polyblend                 "0"
-v_contentblend               "1"
+gl_powerupshells             "1"
```

Comment:
remove gl_powerupshells, or set to 0?  Don't think 'faithful' would have powerup shells

Reply:
No worries.